examples/README.md

```diff
@@ -8,6 +8,7 @@ There are currently two methods available for each solver class [DL-CCVM, MF-CCV
 - `ccvm_boxqp_dl.py`: Solves DL-CCVM using the Adam algorithm, without plotting. The original solver can be used by removing the `algorithm_parameters` argument.
 - `ccvm_boxqp_mf.py`: Solves MF-CCVM using the Adam algorithm, without plotting. The original solver can be used by removing the `algorithm_parameters` argument.
 - `langevin_boxqp.py`: Solves Langevin equation using the Adam algorithm, without plotting. The original solver can be used by removing the `algorithm_parameters` argument.
+- `pumped_langevin_boxqp.py`: Solves pumped Langevin equation by either Adam algorithm or original solver with a use of `algorithm_parameters` argument.
```

Comment:
This is not as clear to me. Maybe write something like this:
```suggestion
- `pumped_langevin_boxqp.py`: Solves the box-constrained programming problem using the pumped Langevin solver using either Adam algorithm for the calculation of the gradient of the objective function or the simple gradient descent method. This choice can be set in the argument of `algorithm_parameters`. 
```

Reply:
Thanks Farhad, the docstring was replaced as suggested.